apps/digitalrain/digital_rain.star

```diff
@@ -101,44 +100,50 @@ def main(config):
     # rand can't assign seed directly, so need to make this a mutable thing
     seed = [seed]
 
+    # get the color; do it this way so that setting the color from the
+    # config doesn't spoil the pseudo-random number sequence
+    color_options = (
+        [i for i in range(COLOR_COUNT)] +
+        [rand(seed, COLOR_COUNT + 1) - 1, rand(seed, COLOR_COUNT)]
+    )
+    color_number = COLOR_NAMES.get(config.get("color"), COLOR_NAMES["random"])
+    if color_number >= 0:
```

Comment:
Is there ever a case this would be less then zero? Or just in case someone tries to through pixlet?

Reply:
The `multicolor` has a value of -1.